bld/CLMBuildNamelist.pm

```diff
@@ -3479,18 +3473,18 @@ sub setup_logic_nitrogen_deposition {
   my ($opts, $nl_flags, $definition, $defaults, $nl) = @_;
 
   #
-  # Nitrogen deposition for bgc=CN
+  # Nitrogen deposition for bgc=CN or fates
```

Comment:
These changes are actually not working right now, so I think this should be a future change. See #2065

Reply:
Since, it's not working it doesn't matter much either way.